packages/x-charts/src/context/CartesianContextProvider.tsx

```diff
@@ -61,43 +68,54 @@ export function CartesianContextProvider({
   const drawingArea = React.useContext(DrawingContext);
 
   const value = React.useMemo(() => {
-    const completedXAxis: DefaultizedAxisConfig = {};
-    const completedYAxis: DefaultizedAxisConfig = {};
+    const axisExtremumCallback = <T extends keyof typeof xExtremumGetters>(
+      acc: ExtremumGetterResult,
+      chartType: T,
+      axis: AxisConfig,
+    ): ExtremumGetterResult => {
+      const getter = xExtremumGetters[chartType];
+      const series = (formattedSeries[chartType]?.series as { [id: string]: ChartSeries<T> }) ?? {};
```

Comment:
You can not use `xExtremumGetters` for both x and y. That's why all the charts in the docs appear empty

```suggestion
      axis: AxisConfig,
      getters: { [T in ChartSeriesType]: ExtremumGetter<T> };
    ): ExtremumGetterResult => {
      const series = (formattedSeries[chartType]?.series as { [id: string]: ChartSeries<T> }) ?? {};
```